### PR TITLE
replace deprecated size props with boxSize in Image component

### LINKED ISSUE #2484: Deprecated props are being used in the Storybook `Image` page.
### Which package has an issue?

@yamada-ui/react

### Package Version

1.4.8

### Description

Deprecated props are being used in the Storybook `Image` page.

Use `boxSize` instead of `size`.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by dev-satoshi**
Closes #2484 

## Description

This Pull Request addresses the deprecation of the `size` props in the `Image` component by replacing it with the `boxSize` props, as per the latest guidelines.

## Current behavior (updates)

The `Image` component currently uses the deprecated `size` props, which may lead to potential issues or warnings in future versions.

## New behavior

The `Image` component now uses the `boxSize` props instead of the deprecated `size` props.
This change aligns with the current best practices and ensures forward compatibility.

## Is this a breaking change (Yes/No):

No

## Additional Information

No additional dependencies or significant changes were introduced.
This is a simple replacement to ensure compatibility with future versions.

### CODE REVIEW

**Comment by dev-satoshi** [comment]
If fixing the tests is within the scope of this issue (#2484), I will proceed with the necessary adjustments right away.

File: .changeset/many-fireants-drop.md
Comment (hirotomoyamada) [inline .changeset/many-fireants-drop.md:1]: Please delete file.

This file is only needed when the package is changed.
It is not needed in this case due to documentation modifications.

Comment (dev-satoshi) [reply .changeset/many-fireants-drop.md:1]: Got it! I've made the changes
[b361cab](https://github.com/yamada-ui/yamada-ui/pull/2533/commits/b361cab57594f205cb9bd00ec88378d4bb6dd3ab)

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**